lib/deck.rb

```diff
@@ -0,0 +1,21 @@
+class Deck
+  attr_reader :cards
+
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    cards.length
+  end
+
+  def cards_in_category(category)
+    category_match = []
+    @cards.each do |card|
```

Comment:
See if you can change `.each` to a different enumerable. I wrote a blog post back when I was in Turing on deciding what enumerable to use: [Removing Each Each](https://medium.com/@anna.noel.lewis/removing-each-each-how-to-refactor-your-ruby-enumerables-fa890aedd08f). I'm kind of terrible at writing, but there's a very nice flow chart towards the end that might be useful. Although in this case, it might be more helpful to ask yourself what the goal is here. You end up with something like "I want to _find all_ cards in the given category."